data-sets-api-server/src/main/java/org/zowe/unix/files/services/zosmf/DeleteUnixFileRunner.java

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
For consistency with other runners rename file to DeleteUnixFileZosmfRunner.java

Reply:
modified added "zosmf" to runner name